float64 signal_strength
float64 frequency
string essid
